pkg/authorization/api/types.go

```diff
@@ -77,6 +78,10 @@ type PolicyRule struct {
 	Resources []string
 	// ResourceNames is an optional white list of names that the rule applies to.  An empty set means that everything is allowed.
 	ResourceNames kutil.StringSet
+	// NonResourceUrls is a set of partial urls that a user should have access to.  *s are allowed, but only as the full, final step in the path
+	// If an action does not match the Verbs, Resources, and ResourceNames of a PolicyRule, then the remainder of the URL (path parts for the request
+	// relative to /{resource}/{name}), is checked against the NonResourceURLs to look for a match.
```

Comment:
ok... I guess I'm forgetting what we talked about... did we end up at remainder URL matching?